src/snowflake/connector/auth/keypair.py

```diff
@@ -63,22 +68,27 @@ def __init__(self, private_key, lifetime_in_seconds: int = LIFETIME):
                 "JWT_CNXN_RETRY_ATTEMPTS", AuthByKeyPair.DEFAULT_JWT_RETRY_ATTEMPTS
             )
         )
-        self._jwt_cnxn_wait_time = timedelta(
+        self._timeout = timedelta(
             seconds=int(
                 os.getenv(
                     "JWT_CNXN_WAIT_TIME", AuthByKeyPair.DEFAULT_JWT_CNXN_WAIT_TIME
                 )
             )
-        )
+        ).seconds
```

Comment:
`timedelta.seconds` only works for timedelta < 1day, if it's greater than 1 day, use `timedelta.total_seconds()`